cliquet/resource.py

```diff
@@ -278,6 +278,9 @@ def __init__(self, request, context):
         logger.bind(collection_id=self.collection.collection_id,
                     collection_timestamp=self.timestamp)
 
+    def get_parent_id(self, request):
```

Comment:
Why passing the request? 

Since this is becoming public API, it deserves a docstring and some tests :)